walkthroughs/1B-protecting-applications-using-rh-sso/walkthrough.adoc

```diff
@@ -0,0 +1,187 @@
+to create an *Identity Provider* in a *Realm* in SSO to enable login
+using social identities such as Google, Twitter, or GitHub. When the user
+chooses to login using an *Identity Provider* a *User* is automatically created
+in the SSO *Realm* with the details from their chosen
+*Identity Provider*.
+
+In this Solution Pattern a *User* will be created manually to keep things
+simple.
+
+. Navigate to the link:{sso-realm-url}[SSO Realm, window="_blank"].
+. Enter the username `{user-username}` and password `password` if prompted.
+. Select the *Users* entry in the side menu.
+. Click *Add user*.
+.. Enter the username `customer`.
+.. Leave other fields with their default values and click *Save*.
+. The `customer` user details should now be displayed with tabs along the top of the UI.
+. Select the *Credentials* tab.
+.. Enter the password `customer-password` in the *New Password* field.
+.. Enter the same password in the *Password Confirmation* field.
+.. Set the *Temporary* toggle to the *OFF* position.
++
+NOTE: If we left *Temporary* in the *ON* position the user would be forced to create a new password when they perform a login.
+.. Click the *Reset Password* button.
+.. When prompted click the *Change password* button in the modal.
+
+[type=verification]
+Select the *Users* item in the side menu. Click the *View all users* button. The `customer` user should be listed.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above. If you are still having issues, contact your administrator.
+
+[time=15]
+== Enabling SSO in the Order Entry System
+
+=== Obtaining the SSO Configuration
+
+To secure an application with SSO a *Client Adapter* is required.
+Various platforms are supported with *Client Adapters*:
+
+. Spring Boot
+. Node.js
+. JBoss EAP
+. Fuse
+. JavaScript (client-side)
+. Servlet Filter
+
+{blank}
+
+The *Order Entry System* is run from a Node.js server, so the Node.js
+`keycloak-connect` adapter is included in the code. The following steps will
+demonstrate how to include a configuration and enable the adapter.
++
+NOTE: Enabling the adapter can vary depending on how an application has been coded. The Order Entry System has been developed to enable SSO protection if a configuration is found in the environment on startup.
+
+. Navigate to the link:{sso-realm-url}[SSO Realm, window="_blank"].
+. Enter the username `{user-username}` and password `password` if prompted. 
+. Select *Clients* from the side menu.
+. Click the `order-entry-system` client that was created earlier.
+. Choose the *Installation* tab.
+. Select *Keycloak OIDC JSON* for *Format Option*.
+. Click the *Download* button to download this as a _keycloak.json_ file.
+
+=== Creating a SSO Config Map Entry
+
+. Login to the link:{openshift-host}/console/[OpenShift Console, window="_blank"].
+. Select the *walkthroughs-1A-integrate-event-and-api-driven-apps* project.
```

Comment:
nitpick, but the actual name is `evals12 - tutorial-web-app-walkthroughs-walkthroughs-1A-integrate-event-and-api-driven-apps`, so maybe we should update this to `<user name> - <location>-walkthroughs-1A-integrate-event-and-api-driven-apps` or maybe just say something like `Selector the project that contains walkthroughs-1A-integrate-event-and-api-driven-apps in the name`?

Reply:
> Selector the project that contains walkthroughs-1A-integrate-event-and-api-driven-apps in the name

Yep, that was my intention. Will fix to make that clear per your suggestion 👍